### DISCUSSION TITLE
Your Personal Account was automatically blocked because its usage exceeds the fair use policy

### DISCUSSION THREAD

**Discussion by voltCore** [vercel/community#2573] [Help]
### Summary

Hello everybody!
I have ban, but i don't no how i can remove it, also i don't know where I can write letter for unblocked myself. Thank you for helping.
<img width="1206" alt="Screenshot 2023-05-24 at 12 01 14" src="https://github.com/vercel/community/assets/97832488/8669ab07-282a-4684-8c2b-8395b5536f31">

**Comment by voltCore** [comment]
Sorry guys. I already resolved my issue. Thanks All

**Reply by arfin-arif** [reply]
How you solved this?
